apply.js

```diff
@@ -55,7 +91,11 @@ function disableAll(disable) {
 function removeStyle(id, doc) {
 	var e = doc.getElementById("stylish-" + id);
 	if (e) {
-		e.parentNode.removeChild(e);
+		delete g_styleElements[e.id];
```

Comment:
What happens if the style element was removed from the doc already (e.g. by the page's own script)? Might it be better to key on the style's ID rather than the style element's ID?